tasks/collect-data/collect-data.yaml

```diff
@@ -104,17 +92,17 @@ spec:
 
         release_result=$(get-resource "release" "${RELEASE}" "{.spec.extraData}")
```

Comment:
is this not becoming data too? Or maybe just not yet because it isn't in the release-service pr?

Reply:
Yep. Fixed.